docs/tutorials/stateful-application/run-stateful-application.md

```diff
@@ -21,50 +21,36 @@ application we will use is MySQL.
 
 {% capture prerequisites %}
 
-* To do this tutorial, you need a Kubernetes cluster, including a running
-  Kubernetes API server. You can use an existing cluster, or you can create a
-  new cluster. One way to create a new cluster is to use
-  [Minikube](/docs/getting-started-guides/minikube).
-
-* You also need to have `kubectl` installed on your local machine, and `kubectl`
-  must be configured to communicate with your Kubernetes API server. This
-  configuration is done automatically if you use Minikube.
-
-* For data persistence we will create a Persistent Volume that
-  references a disk in your
-  environment. See
-  [here](/docs/user-guide/persistent-volumes/#types-of-persistent-volumes) for
-  the types of environments supported. This Tutorial will demonstrate
-  `GCEPersistentDisk` but any type will work. `GCEPersistentDisk`
-  volumes only work on Google Compute Engine.
+* {% include task-tutorial-prereqs.md %}
```

Comment:
Steve, The last bullet is important for this sample, I know it is covered in the first step of the tutorial, but I think it is important to call out in the prereqs.